geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -265,30 +292,27 @@ long zcount(SortedSetScoreRangeOptions rangeOptions) {
     return maxIndex - minIndex;
   }
 
-  byte[] zincrby(Region<RedisKey, RedisData> region, RedisKey key, byte[] increment,
-      byte[] member) {
+  double zincrby(Region<RedisKey, RedisData> region, RedisKey key, double incr, byte[] member) {
```

Comment:
For consistency, could either this method return a `byte[]`, or could other methods in this class that return a score also return it as `double`? The former is probably the most consistent and simplest, given we have range commands that return interleaved member and score `byte[]` if "WITHSCORES" is specified and it wouldn't be sensible to change those to return `double` rather than `byte[]` for the score now.